packages/react/src/components/PaginationNav/PaginationNav.tsx

```diff
@@ -504,8 +511,12 @@ const PaginationNav = React.forwardRef<HTMLElement, PaginationNavProps>(
 
     const startOffset = itemsDisplayedOnPage <= 4 && currentPage > 1 ? 0 : 1;
 
+    const accessibilityLabel = ariaLabelledBy
+      ? { 'aria-labelledby': ariaLabelledBy }
+      : { 'aria-label': ariaLabel ?? 'pagination' };
+
     return (
-      <nav className={classNames} ref={ref} {...rest} aria-label="pagination">
+      <nav className={classNames} ref={ref} {...rest} {...accessibilityLabel}>
```

Comment:
```suggestion
      <nav className={classNames} ref={ref} {...rest}>
```

Since the `nav` isn't required to have an accessible name, what if these weren't part of the proptypes/interface and instead just passed through `...rest`?